docs/multi-homing.md

```diff
@@ -0,0 +1,106 @@
+# Multi-homing
+A K8s pod with more than one network interface is said to be multi-homed. The
+[Network Plumbing Working Group](https://github.com/k8snetworkplumbingwg/multi-net-spec)
+has put forward a [standard](https://github.com/k8snetworkplumbingwg/multi-net-spec)
+describing how to specify the configurations for additional network interfaces.
+
+There are several delegating plugins or meta-plugins (Multus, Genie) implementing this standard.
+
+After a pod is scheduled on a particular Kubernetes node, kubelet will invoke
+the delegating plugin to prepare the pod for networking. This meta-plugin will
+then invoke the CNI responsible for setting up the pod's default cluster
+network, and afterwards it iterates the list of additional attachments on the
+pod (yes, these plugins **must** be able to access the Kubernetes API...),
+invoking the corresponding delegate CNI implementing the logic to attach the pod
+to that particular network.
+
+## Configuring secondary networks
+To allow pods to have multiple network interfaces, the user must provide the
+configurations specifying how to connect to these networks; these
+configurations are defined in a CRD named `NetworkAttachmentDefinition`.
+
+Below you will find example attachment configurations for each of the current
+topologies OVN-K allows for secondary networks.
+
+**NOTE**: currently, all the secondary networks **only** allow for east/west
+traffic.
+
+### Routed - layer 3 - topology
+This topology is a simplification of the topology for the cluster default
+network - but without egress.
+
+There is a logical switch per node - each with a different subnet - and a
+router interconnecting all the logical switches.
+
+The following net-attach-def configures the attachment to a routed secondary
+network.
+
+```yaml
+apiVersion: k8s.cni.cncf.io/v1
+kind: NetworkAttachmentDefinition
+metadata:
+  name: l3-network
+  namespace: left-one
+spec:
+  config: |2
+    {
+            "cniVersion": "0.3.1",
+            "name": "l3-network",
+            "type": "ovn-k8s-cni-overlay",
+            "topology":"layer3",
+            "subnets": "10.128.0.0/16/24",
```

Comment:
indicate this support multiple subnets that is comma-separated?

Reply:
Done.